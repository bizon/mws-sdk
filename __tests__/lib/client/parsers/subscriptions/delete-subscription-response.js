const parseXml = require('../../../../../lib/client/parsers')
const parseDeleteSubscriptionResponse = require('../../../../../lib/client/parsers/subscriptions/delete-subscription-response')

describe('lib.client.parsers.subscriptions.delete-subscription-response', () => {
  it('should parse the DeleteSubscriptionResponse example response from MWS doc', () => {
    const doc = parseXml(
      `<DeleteSubscriptionResponse xmlns="http://mws.amazonservices.com/schema/Subscriptions/2013-07-01">
         <DeleteSubscriptionResult />
         <ResponseMetadata>
           <RequestId>2d7db8a1-8974-4541-9c9b-f882dEXAMPLE</RequestId>
         </ResponseMetadata>
       </DeleteSubscriptionResponse>`,
    )

    const res = parseDeleteSubscriptionResponse(
      '/subscriptions:DeleteSubscriptionResponse',
      doc,
    )

    expect(res).toMatchSnapshot()
  })
})
