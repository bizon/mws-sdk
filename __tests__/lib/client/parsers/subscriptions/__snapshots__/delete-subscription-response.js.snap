// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lib.client.parsers.subscriptions.delete-subscription-response should parse the DeleteSubscriptionResponse example response from MWS doc 1`] = `
{
  "deleteSubscriptionResult": {},
  "responseMetadata": {
    "requestId": "2d7db8a1-8974-4541-9c9b-f882dEXAMPLE",
  },
}
`;
